src/frontend/web_application/src/components/MessageList/components/Message/index.jsx

```diff
@@ -100,7 +108,7 @@ class Message extends Component {
 
   render() {
     const { message, __ } = this.props;
-    const author = message.participants.find(participant => participant.type === 'from');
+    const author = message.participants.find(participant => participant.type === 'From');
```

Comment:
This information would probably be better of in `mapStateToProps` via a selector?